static/docs/commands-reference/update.md

```diff
@@ -37,3 +37,32 @@ their DVC-files (via fields `lock: true` and `rev_lock`). For a locked stage,
   problems arise, otherwise 1.
 
 - `-v`, `--verbose` - displays detailed tracing information.
+
+## Example
+
+Let's first import a data artifact from our
+[get started example repo](https://github.com/iterative/example-get-started):
+
+```dvc
+$ dvc import git@github.com:iterative/example-get-started model.pkl
+Importing 'model.pkl (git@github.com:iterative/example-get-started)' -> 'model.pkl'
+...
+Saving information to 'model.pkl.dvc'.
+...
+```
+
+As DVC mentions, the import stage (DVC-file) `model.pkl.dvc` is created. this
+[stage file](/doc/commands-reference/run) is locked by default though, so to
+[reproduce](/doc/commands-reference/repro) it, we would need to run `dvc unlock`
+on it first, then `dvc repro` (and `dvc lock` again). Let's just run
+`dvc update` on it instead:
+
+```dvc
+$ dvc update model.pkl.dvc
+Stage 'model.pkl.dvc' didn't change.
+Output 'model.pkl' didn't change. Skipping saving.
+Saving information to 'model.pkl.dvc'.
+```
+
+This time nothing has changed (since our Get Started sample repo is pretty
```

Comment:
link to Get started repo, also sometimes you use it all lower case

Reply:
Hmmm. It's already linked at the beginning of the example (link 44 right now). But I agree this line is strange, changed in 20b8f0d